waku/waku.md

```diff
@@ -114,6 +114,8 @@ topic-node-enabled = BIT
 
 confirmations-enabled = BIT
 
+light-node = BIT
```

Comment:
`topic-node-enabled = BIT` above still needs to be removed. AFAIK this is not used in any of the packets.